src/yunohost/vendor/acme_tiny/acme_tiny.py

```diff
@@ -95,7 +82,7 @@ def _send_signed_request(url, payload):
     log.info("Registering account...")
     code, result = _send_signed_request(CA + "/acme/new-reg", {
         "resource": "new-reg",
-        "agreement": "https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf",
+        "agreement": json.loads(urlopen(CA + "/directory").read().decode('utf8'))['meta']['terms-of-service'],
```

Comment:
Not sur to be the right person to review this change.

- lot of new files, is this because you decide to use git subtree ?
- the main change (appart from pep8) in acme_tiny.py is here, I don't understand this change but it seems legit

Reply:
> lot of new files, is this because you decide to use git subtree ?

Exactly.

>the main change (appart from pep8) in acme_tiny.py is here, I don't understand this change but it seems legit

This change happens [upstream](https://github.com/diafygi/acme-tiny/commit/c4940d229a296b7643e86d2e9ab31a6e0099ba71) to fix this problem.

Reply:
LGTM